### PR TITLE
fix: include meta for replace pin

### PR DESCRIPTION
**Pull request by GaryHomewood**

### CODE REVIEW

File: packages/api/src/pins.js [hunk 1]
```diff
@@ -144,7 +144,8 @@ async function createPin (normalizedCid, pinData, authTokenId, env, ctx) {
 
   await env.cluster.pin(cid, {
     name: pinName,
-    origins
+    origins,
+    metadata: pinMeta
```
Comment (alanshaw) [inline packages/api/src/pins.js:148]: We can't send metadata to cluster it is not multi-tenant - what happens if someone else pins the same CID?

**Review by alanshaw — CHANGES_REQUESTED** [review]

File: packages/api/src/pins.js (hunk 1 above)
Comment (GaryHomewood) [reply packages/api/src/pins.js:148]: It looks like its passed in NFT `pins-add`. Is that different?

Comment (alanshaw) [reply packages/api/src/pins.js:148]: Nope, it should not be sent there either. Can you send a PR to fix? 🙏

Comment (GaryHomewood) [reply packages/api/src/pins.js]: Ok cool. Removed it here, created an issue on NFT

[https://github.com/nftstorage/nft.storage/issues/1207](https://github.com/nftstorage/nft.storage/issues/1207)

**Review by alanshaw — APPROVED** [review]

**Pull request merged**